### PR TITLE
Implement getter and setter for MaxSize

### PR DESCRIPTION
**Pull request by igsr5**
https://github.com/h2non/bimg/issues/404 
> I would like a way to change MaxSize by myself.

Added getter and setter to allow you to change maxSize yourself.

### CODE REVIEW

File: vips.go [hunk 1]
```diff
@@ -581,7 +581,7 @@ func vipsSmartCrop(image *C.VipsImage, width, height int) (*C.VipsImage, error)
 	var buf *C.VipsImage
 	defer C.g_object_unref(C.gpointer(image))
 
-	if width > MaxSize || height > MaxSize {
+	if width > MaxSize() || height > MaxSize() {
```
Comment (h2non) [inline vips.go:584]: You can use `maxSize` variable instead of function call.

File: options.go [hunk 2]
```diff
@@ -5,14 +5,34 @@ package bimg
...
+// maxSize defines maximum pixels width or height supported.
+func MaxSize() int {
+	return maxSize
+}
+
+// SetMaxSize sets maxSize.
+// maxSize defines maximum pixels width or height supported.
+func SetMaxsize(s int) error {
+	if s <= 0 {
+		return fmt.Errorf("invalid size.A size must be 0.")
```
Comment (h2non) [inline options.go:28]: Change error to `Size must be higher than zero.`

File: options.go [hunk 3]
```diff
@@ -5,14 +5,34 @@ package bimg
...
 */
 import "C"
+import "fmt"
 
 const (
 	// Quality defines the default JPEG quality to be used.
 	Quality = 75
-	// MaxSize defines the maximum pixels width or height supported.
-	MaxSize = 16383
+	// maxSize defines the maximum pixels width or height supported.
```
Comment (h2non) [inline options.go:13]: Delete this comment.

File: options.go [hunk 4]
```diff
@@ -5,14 +5,34 @@ package bimg
...
 )
 
+var maxSize = 16383
+
+// MaxSize returns maxSize.
+// maxSize defines maximum pixels width or height supported.
```
Comment (h2non) [inline options.go:19]: Move this comment to line `15`.

File: options.go [hunk 5]
```diff
@@ -5,14 +5,34 @@ package bimg
...
+func MaxSize() int {
+	return maxSize
+}
+
+// SetMaxSize sets maxSize.
+// maxSize defines maximum pixels width or height supported.
```
Comment (h2non) [inline options.go:25]: Delete this comment as it is redundant.

File: options.go (hunk 2 above)
Comment (h2non) [reply options.go:28]: Using `fmt` here does not add any benefits, use `errors.New()` instead.

**Review by h2non — COMMENTED** [review]
Thanks for the PR. Please address my comments and I will merge it.

**Comment by igsr5** [comment]
Thanks for the review comment.

Sorry, I was in a hurry and made a lot of ordinary mistakes...
I have fixed them.

**Comment by h2non** [comment]
Thanks! Version `1.1.9` is now available with this feature.

**Pull request merged**